cli/cmd/source/add.go

```diff
@@ -0,0 +1,24 @@
+package source
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// addCmd represents the add command
+func AddCmd() *cobra.Command {
+	var sourcePath string
+	var addCmd = &cobra.Command{
+		Use:   "add",
+		Short: "Add source to the project",
+		Long:  `Add source to the project`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			fmt.Println("add called")
+			return nil
+		},
+	}
+
+	addCmd.Flags().StringVarP(&sourcePath, "source-path", "p", ".", "Source path for Rill")
```

Comment:
This should be a required arg, probably a positional arg and not a flag, e.g. `rill source add /path/to/file.csv`

Reply:
Make sense, keep it as args with check of MinimumNArgs(1) so that it can support multiple if required in future